src/components/pages/career-details/career-details.tsx

```diff
@@ -60,7 +65,9 @@ export const CareerDetails = ({
 
       <Form
         company_id="41230"
-        recruiting_channel_id={personioChannel ?? DEFAULT_CHANNEL_ID}
+        recruiting_channel_id={
+          utmChannel ?? personioChannel ?? DEFAULT_CHANNEL_ID
```

Comment:
To keep logic out of the JSX, I would also put this check into its own variable. This way, you have all related code to the `recruitingChannelId` in one place.

Reply:
👍